presto-hive/src/main/java/com/facebook/presto/hive/HiveWriteUtils.java

```diff
@@ -456,8 +467,8 @@ private static boolean isDirectory(String user, HdfsEnvironment hdfsEnvironment,
 
     public static Path createTemporaryPath(String user, HdfsEnvironment hdfsEnvironment, Path targetPath)
     {
-        // use a per-user temporary directory to avoid permission problems
```

Comment:
I'd like to keep the original comment to indicate why we use a per-user directory.